src/lib/ggpo/network/udp.cpp

```diff
@@ -83,7 +90,11 @@ Udp::OnLoopPoll(void *cookie)
 {
    uint8          recv_buf[MAX_UDP_PACKET_SIZE];
    sockaddr_in    recv_addr;
+#ifdef _WIN32
```

Comment:
I think the definition of `socklen_t` is in `WS2tcpip.h`.  Let's just include that in `platform_window.h` and remove these ifdefs.